src/template/package.json

```diff
@@ -9,6 +9,7 @@
     "build": "tsc",
     "watch": "tsc -w",
     "test": "jest",
+    "test:dev": "jest --watch",
```

Comment:
`yarn test:dev` is being run by the [start script](https://github.com/guardian/cdk-cli/blob/afcb541db4decae167956e81fdebc0b5ed7717f2/src/template/script/start).